core/src/main/java/com/scalar/db/io/Key.java

```diff
@@ -586,6 +586,15 @@ public static Key ofBlob(String columnName, ByteBuffer value) {
     return new Key(columnName, value);
   }
 
+  /**
+   * Create an empty {@code Key} object
+   *
+   * @return an empty {@code Key} object
+   */
+  public static Key of() {
```

Comment:
Would `Key.empty()` be appropriate ?

Reply:
In `ImmutableXXX` in Guava, we can create empty collections with `of()` without arguments. I followed this naming. What do you think?

Reply:
All right. If it's a well known usage, that's good.